ci-operator/step-registry/ibmcloud/provision/kms-key/ibmcloud-provision-kms-key-commands.sh

```diff
@@ -24,6 +24,29 @@ function create_resource_group() {
     "${IBMCLOUD_CLI}" target -g ${rg} || return 1
 }
 
+function createKey() {
+    local instance_name=$1
+    local region=$2
+    ibmcloud resource service-instance-create ${instance_name} kms tiered-pricing ${region}
+    id=$(ibmcloud resource service-instance ${instance_name} --id -q | awk '{print $2}')
+   
+    key_name="${instance_name}-key"
+    key_meterial=$(openssl rand -base64 32)
+    tmpFile=$(mktemp)
+    ibmcloud kp key create ${key_name} -k ${key_meterial} -i ${id} -o json > ${tmpFile}
+
+    keyCRN=$(jq -r .crn $tmpFile)
+    keyID=$(jq -r .id $tmpFile)
+
+    keyInfo=$(jq -n \
```

Comment:
If you want to save these info, I suggest to add one more parameter as a filename to save these info directly.

Reply:
I've tried with a temp file, then reused it in the several createKey, but always failed to get the expected value, so I changed to the global string.

Reply:
```
[root@preserve-jialiu ~]# cat test.json 
{
  "id": "aa",
  "keyID": "bb",
  "keyCRN": "cc"
}
[root@preserve-jialiu ~]# keyInfo="$(cat test.json)"
[root@preserve-jialiu ~]# cat my.json
{
  "resource_group": "xx"
}
[root@preserve-jialiu ~]# jq --argjson idarg "$keyInfo" '. += {"master": $idarg}' my.json
{
  "resource_group": "xx",
  "master": {
    "id": "aa",
    "keyID": "bb",
    "keyCRN": "cc"
  }
}
```
I did not see any problem here.